src/chrome/content/rules/BASt.de.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+	Invalid certificate:
+		bisstra-upload.bast.de
+		itzeb.bast.de
+
+	Refused:
+		deufrako.bast.de
+		ripcord.bast.de
+		verdi.bast.de
+		visustrplanung.bast.de
+
+	Timeout:
+		ravel.bast.de
```

Comment:
This subdomain times out over both HTTP and HTTPS. I think we can completely remove it